oslabs-beta/KataLog
# Cf/http output

##Overview
**Issue Type**
- []Tech debt

**Description**

Feature: The `Projects` page now asks a user to select a project (`selectedProject`) and makes an API call to the database to retrieve logs to associated with that specific project.

Bug: Pagination in `LogsAsComponent.tsx` is trying to render a React Object when any page after page 1 is selected. May not be an issue when this moves to the `Logs` page.

**Steps to Reproduce Bug/Validate Feature/Confirm Tech Debt Fix**
- Go to ‘Projects’
- Click on the dropdown menu and select a project
- Scroll down to the logs component
- Select any page after page 1
- See error.

**Previous Behavior**
Logs were being rendered via a fetch request to a file. Now the application fetches from the database. The `LogsAsComponent.tsx` (which did not exist previously) does not render logs past page 1.

**Expected Behavior**
Any pagination should render the logs fetched after page 1

**How Has This Been Tested**
See "Steps to Reproduce Bug/Validate Feature/Confirm Tech Debt Fix"

**Screenshots & Videos**
N/A

**Additional Context**
As stated, may not be an issue when this moves to the `Logs` page.
